utilities/src/main/java/io/syndesis/qe/bdd/validation/MetricsValidationSteps.java

```diff
@@ -0,0 +1,60 @@
+package io.syndesis.qe.bdd.validation;
+
+import org.assertj.core.api.Assertions;
+
+import java.security.GeneralSecurityException;
+import java.util.Date;
+import java.util.List;
+
+import cucumber.api.java.en.Then;
+import io.syndesis.model.integration.Integration;
+import io.syndesis.model.metrics.IntegrationMetricsSummary;
+import io.syndesis.qe.endpoints.IntegrationsEndpoint;
+import io.syndesis.qe.endpoints.MetricsEndpoint;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class MetricsValidationSteps {
+
+	private IntegrationsEndpoint integrationsEndpoint;
+	private MetricsEndpoint metricsEndpoint;
+
+	public MetricsValidationSteps () {
+		try {
+			integrationsEndpoint = new IntegrationsEndpoint();
+			metricsEndpoint = new MetricsEndpoint();
+		} catch (GeneralSecurityException e) {
+			Assertions.fail("Error: " + e);
+		}
+
+
+	}
+	@Then("^validate that number of all messages through integration \"([^\"]*)\" is greater than \"([^\"]*)\", period in ms: \"([^\"]*)\"$")
+	public void validateThatNumberOfAllMessagesOfIntegrationIsGreaterThanPeriodInMs(String integrationName, Integer nr, Integer ms) throws InterruptedException {
```

Comment:
I suppose you check all the available values that are provided by metrics endpoint, right?

Reply:
I'm going to (asap when suitable syndedis instance is available)